cli/flox/doc/manifest.toml.md

```diff
@@ -0,0 +1,310 @@
+    Multiple package groups may resolve to the same catalog revision.
+    Package groups may be upgraded as a unit,
+    ensuring that the group continues to work together.
+    See [`flox-upgrade(1)`](./flox-upgrade.md) for more details on how groups
+    and packages interact during upgrades.
+
+`version`
+:   Requires that the package match either an exact version or a semver range.
+    When the first character of the `version` string is '=' the version must be
+    an exact match,
+    otherwise the `version` string is matched as a semver range.
+    Versions that don't conform to semver must be specified with '='.
+    
+    The semantic version can be specified with the typical qualifiers such as
+    `^`, `>=`, etc.
+    Semantic versions that do not specify all three fields
+    (`MAJOR.MINOR.PATCH`) will treat the unspecified fields as wildcards.
+    This instructs Flox to find the latest versions for those fields.
+    For example `version = "1.2"` would select the latest version in the
+    `1.2.X` series.
+
+    This option is mutually exclusive with the `semver` option.
+
+`semver`
+:   This option is similar to `version` except it _only_ allows semantic
+    versions.
+
+`systems`
+:   A list of systems on which to install this package.
+    When omitted this defaults to the same systems that the environment
+    specifies that it supports.
+
+`pkg-path`
+:   The abbreviated location of a package within a catalog.
+    A pkg-path is a sequence of one or more attributes joined by a delimiter.
+    For example, both `ripgrep` and `python310Packages.pip` are pkg-paths.
+    A pkg-path that contains more than one attribute can be represented as
+    either a single string that contains a '.'-delimited sequence of the
+    attributes,
+    or it can be represented as a TOML array of strings where each string is
+    an attribute.
+    For example, both `"python310Packages.pip"`
+    and `["python310Packages", "pip"]` are equivalent for the `pkg-path`
+    option.
+
+    This option is mutually exclusive with `abs-path`.
+
+`abs-path`
+:   The fully-qualified location of a package within a catalog.
+    For the `ripgrep` package in the Flox base catalog for an `x86_64-linux`
+    system this would be `legacyPackages.x86_64-linux.ripgrep`.
+    Note that "legacyPackages" has nothing to do packages being out of date,
+    and instead has to do with internal Flox implementation details.
+    You should rarely ever need this option and should instead prefer the
+    `pkg-path` option.
+
+`priority`
+:   A priority used to resolve file conflicts
+    The `/bin`, `/man`, `/include`, etc directories from all packages in the
+    environment are merged when building the environment.
+    Packages that provide the exact same file e.g. `/bin/foo` cause a file
+    conflict that prevents the environment from building since it's ambiguous
+    which of the packages should eventually provide the `/bin/foo` binary.
```

Comment:
nit: this sentence is a chonker, might want to split it up

Reply:
Reworded it so it's more concise.